packages/atomic/src/components/atomic-tab/atomic-tab.tsx

```diff
@@ -22,7 +22,16 @@ export class AtomicTab implements InitializableComponent {
   @BindStateToController('tab') @State() private tabState!: TabState;
   @State() public error!: Error;
 
+  /**
+   * The advanced expression or filter that the Tab should add to any outgoing query.
```

Comment:
Tabs actually use the constant expression. Tabs won't be part of the first official release though, they will be revisited later.

Reply:
Oh I see. I fixed the string and the doc will be ready for when you revisit tabs :) 
And thanks for the review!